src/main/kotlin/com/autonomousapps/model/internal/Source.kt

```diff
@@ -162,7 +161,7 @@ internal data class AndroidResSource(
        *
        * @see <a href="https://developer.android.com/guide/topics/resources/providing-resources#ResourcesFromXml">Accessing resources from XML</a>
        */
-      private val TYPE_REGEX = Regex("""@(\w+:)?(?<type>\w+)/(\w+)""")
+      private val TYPE_REGEX = Regex("""@(\w+:)?(?<type>\w+)/([\w.]+)""")
```

Comment:
`.` are actually used for styles/themes, otherwise it breaks `detects theme set in manifest` and `detects theme set in manifest on Activity` tests.

Reply:
I also added a unit test for this.